sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/StringExpressionsSuite.scala

```diff
@@ -520,6 +520,10 @@ class StringExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
     withSQLConf(SQLConf.CHUNK_BASE64_STRING_ENABLED.key -> "true") {
       checkEvaluation(Base64(Literal(longString.getBytes)), chunkEncoded)
     }
+
+    // check if unbase64 works well for chunked and non-chunked encoded strings
+    checkEvaluation(StringDecode(UnBase64(Literal(encoded)), Literal("utf-8")), longString)
+    checkEvaluation(StringDecode(UnBase64(Literal(chunkEncoded)), Literal("utf-8")), longString)
```

Comment:
I added this case to confirm that the chunkEncoded string can be decoded correctly. cc @cloud-fan